src/store/actions/getApy.ts

```diff
@@ -0,0 +1,13 @@
+import { ActionContext } from '..';
```

Comment:
Put these actions into separate folder? OR export all of them from same file? `earn.ts`

Reply:
Resolved